projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -1914,6 +1915,37 @@ function wpcom_launchpad_mark_verify_email_complete( $user_id ) {
 }
 add_action( 'wpcom_email_verification_complete', 'wpcom_launchpad_mark_verify_email_complete' );
 
+/**
+ * Callback to determine email verification visibility.
+ * This uses a heuristic to determine whether it's likely that the user needed to
+ * verify their email at the time the site was created. That way we're not showing
+ * the completed task to users who were verified before creating this site.
+ *
+ * @param bool  $is_visible Whether the task should be visible.
+ * @param array $data Additional data.
+ * @return bool True when the email verification task should be visible.
+ */
+function wpcom_launchpad_is_email_task_visible( $is_visible, $data ) {
+	if ( ! isset( $data ) || ! isset( $data['launchpad_context'] ) || $data['launchpad_context'] !== 'customer-home-treatment-cumulative' ) {
```

Comment:
I tried logging `$data['launchpad_context']` but didn't get any data back. Haven't looked too deep into this yet.